lib/parity/backup.rb

```diff
@@ -53,10 +53,15 @@ def download_remote_backup
     def restore_from_local_temp_backup
       Kernel.system(
         "pg_restore tmp/latest.backup --verbose --clean --no-acl --no-owner "\
-          "-d #{development_db} #{additional_args}",
+          "--dbname #{development_db} --jobs `#{number_of_cores}` "\
+          "#{additional_args}",
       )
     end
 
+    def number_of_cores
+      "getconf _NPROCESSORS_ONLN"
+    end
```

Comment:
In that case, you may be interested in https://github.com/puppetlabs/facter

`Facter.value('processors')['count']`